internal/service/credentials/data_source_credential_type_test.go

```diff
@@ -30,12 +31,16 @@ func TestAccCredentialTypeDataSource_ByIDFull(t *testing.T) {
 					resource.TestCheckResourceAttrSet(dataSourceFullName, "id"),
 					resource.TestCheckResourceAttrSet(dataSourceFullName, "environment_id"),
 					resource.TestCheckResourceAttrSet(dataSourceFullName, "credential_type_id"),
+					resource.TestCheckResourceAttrSet(dataSourceFullName, "issuer_id"),
```

Comment:
Could these tests check the P1 ID regex?  So that we don't end up with a blank string passing the tests

Reply:
Resolved